Hotfix animal emblem ranks

LeroyAndrade opened the pull request:
+ animal emblems shown
+ animal title shown

LeroyAndrade committed:
Hotfix animal emblem ranks

Animal title & logo now properly shown.

LeroyAndrade committed:
deleted console.log

cipherxof commented on src/system/utility.tsx:
```diff
@@ -139,6 +146,63 @@ export const getRankId = (name: string) => {
+
+// PlusMinus temporary hotfix
+//below
+export const arrayAnimalRanks = [
```
We don't really need a lookup table for the rank images. I'll just store the ranks on the server using the ID instead.

cipherxof commented on src/pages/RankGuide.tsx:
```diff
@@ -469,5 +507,32 @@ export default (): React.ReactNode => {
-};
+  )};
+
+// for (const rank of ranks) {
```
Remove all of the unused code.

cipherxof commented on src/pages/RankGuide.tsx:
```diff
@@ -449,16 +483,20 @@ export default (): React.ReactNode => {
 
-  for (const rank of ranks) {
+  let countArrayAnimalRanks=1;
+  const pathToImg = `https://mgo2pc.com/static/media/emblem/${arrayAnimalRanks[countArrayAnimalRanks][1]}.png`;
```
This variable is unused.

cipherxof commented on src/pages/RankGuide.tsx:
```diff
@@ -1,20 +1,26 @@
+import { getRankId, getRankPreview, getRankImage, arrayAnimalRanks } from '@/system/utility';
 import { PageContainer } from '@ant-design/pro-layout';
-import { Card, Typography } from 'antd';
+import { Card, Descriptions, Typography } from 'antd';
```
Remove unused imports.

cipherxof commented on src/pages/RankGuide.tsx:
```diff
@@ -1,20 +1,26 @@
-import { Card, Typography } from 'antd';
+import { Card, Descriptions, Typography } from 'antd';
 import React from 'react';
+import {useState}  from 'react';
```
unused

cipherxof commented on src/system/utility.tsx:
```diff
@@ -68,7 +68,14 @@ export const logoutUser = () => {
 
 export const getRankPreview = (rank: number) => {
-  return `/static/media/emblem/${rank}.png`;
+  return `https://mgo2pc.com/static/media/emblem/${rank}.png`;
```
should be using relative paths

cipherxof reviewed (changes requested):
In general, I would remove all of your changes except for where you added the ID to the rank images. From there, I'll store the images on the server using the ID which is the simplest solution.

LeroyAndrade committed:
save current state

LeroyAndrade committed:
Hotfix animal emblem ranks

Todo: set absolute path to relative path of document RankGuide - item's don't exist in opensource project - won't guesss

LeroyAndrade committed:
Hotfix animal emblem ranks

Todo: set absolute path to relative path of document RankGuide - item's don't exist in opensource project - won't guesss

LeroyAndrade commented:
Thank you for the feedback.
As of now only the variable containing the absolute path should be edited to the relative path (RankGuide.tsx -> variable: pathToImgAndShowRankEmblem on line #493.
Other then that, all of your requests have been honored and executed.

LeroyAndrade committed:
Deleted the static path

All request of the Pull request have been honored and executed.

cipherxof commented:
Seems good 👍
